### ISSUE TITLE
RCTConvert+AirMap.h module not found react native maps

### ISSUE THREAD

**Issue by rupam999** [react-native-maps/react-native-maps#4552]
### Summary

**I was getting the below error**
RCTConvert+AirMap.h module not found react native maps

So, I added the below lines in my pod file

```
$static_framework += [
    'react-native-maps',
    'react-native-google-maps',
    'Google-Maps-iOS-Utils',
    'GoogleMaps',
    'RNPermissions',
    'Permission-LocationWhenInUse',
    'Permission-Notifications',
    'react-native-paste-input',
    'vision-camera-code-scanner',
    'VisionCamera'
  ]

pre_install do |installer|
    Pod::Installer::Xcode::TargetValidator.send(:define_method, :verify_no_static_framework_transitive_dependencies) {}
        installer.pod_targets.each do |pod|
            if $static_framework.include?(pod.name)
                def pod.build_type;
                Pod::BuildType.static_library # >= 1.9
            end
        end
    end
  end
```

**After that I am getting the below error**

Any Idea how solve this issue

error Failed to launch the app on simulator, An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=3):
Application launch for 'org.reactjs.native.example.BiroCabs' did not return a valid pid nor a launch error.
No such process

### Reproducible sample code

```TSX
require_relative '../node_modules/react-native/scripts/react_native_pods'
require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'

$RNFirebaseAsStaticFramework = true

platform :ios, '13.0'
install! 'cocoapods', :deterministic_uuids => false
use_frameworks! :linkage => :static

target 'App' do
  $static_framework = []
  config = use_native_modules!

  # Flags change depending on the env values.
  flags = get_default_flags()

  # React Native Maps dependencies
  rn_maps_path = '../node_modules/react-native-maps'
  pod 'react-native-google-maps', :path => rn_maps_path

  $static_framework += [
    'react-native-maps',
    'react-native-google-maps',
    'Google-Maps-iOS-Utils',
    'GoogleMaps',
    'RNPermissions',
    'Permission-LocationWhenInUse',
    'Permission-Notifications',
    'react-native-paste-input',
    'vision-camera-code-scanner',
    'VisionCamera'
  ]

  use_react_native!(
    :path => config[:reactNativePath],
    # Hermes is now enabled by default. Disable by setting this flag to false.
    # Upcoming versions of React Native may rely on get_default_flags(), but
    # we make it explicit here to aid in the React Native upgrade process.
    # :hermes_enabled => true,
    # :fabric_enabled => flags[:fabric_enabled],
    # Enables Flipper.
    #
    # Note that if you have use_frameworks! enabled, Flipper will not work and
    # you should disable the next line.
    # :flipper_configuration => FlipperConfiguration.enabled,
    # An absolute path to your application root.
    :app_path => "#{Pod::Config.instance.installation_root}/.."
  )

  target 'AppTests' do
    inherit! :complete
    # Pods for testing
  end

  # ****** THIS IS THE MAGIC ******
  pre_install do |installer|
    Pod::Installer::Xcode::TargetValidator.send(:define_method,  :verify_no_static_framework_transitive_dependencies) {}
        installer.pod_targets.each do |pod|
            if $static_framework.include?(pod.name)
                def pod.build_type;
                Pod::BuildType.static_library # >= 1.9
            end
        end
    end
  end

  post_install do |installer|
    react_native_post_install(
      installer,
      # Set `mac_catalyst_enabled` to `true` in order to apply patches
      # necessary for Mac Catalyst builds
      :mac_catalyst_enabled => false
    )
    __apply_Xcode_12_5_M1_post_install_workaround(installer)
    installer.pods_project.build_configurations.each do |config|
      config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
    end
  end
end
```

### Steps to reproduce

Add react-native-maps

### Expected result

Should run

### Actual result

Error

### React Native Maps Version

1.3.2

### What platforms are you seeing the problem on?

iOS (Google Maps)

### React Native Version

0.70.6

### What version of Expo are you using?

Not using Expo

### Device(s)

Iphone 14

**Comment by sofian-sunny** [comment]
I am also facing same issue when using  use_frameworks! :linkage => :static
 without use_frameworks! :linkage => :static it is working fine but for firebase integration use_frameworks! :linkage => :static
 must be used.

**Comment by koloboid** [comment]
Same here. Has anyone found a solution?

**Comment by yousrasd** [comment]
@rupam999 are you using a mac with M1?

**Comment by hasnainbaloch** [comment]
same issue

**Comment by Hemistone** [comment]
Same issue here. I cannot fix this error even disabling hermes since my other package configurations are hermes-centered😂

**Comment by saxahan** [comment]
Still same

**Comment by Bilal-Abdeen** [comment]
Check out my comment on  [#3597](https://github.com/react-native-maps/react-native-maps/issues/3597#issuecomment-1368291121)

**Comment by japivonka** [comment]
Still same

**Comment by Bilal-Abdeen** [comment]
> 

Most probably, you will need to submit the following command after applying the changes  to the Podfile: 
`npx pod-install`

**Comment by Hemistone** [comment]
@Bilal-Abdeen Thank you for your advice. 

I also solved the situation with hermes enabled. 💪
However, I was able to solve with several more steps additional to your advice. After configuring podfile [as you mentioned](https://github.com/react-native-maps/react-native-maps/pull/4578/files), I had to add custom changes on the iOS import codes by [patch-packages](https://www.npmjs.com/package/patch-package) like this.

Without this patch-packages changes, I kept getting build errors.



`./patches/react-native-maps+1.3.2.patch`
```patch
diff --git a/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMap.m b/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMap.m
index 8a4215c..555f03c 100644
--- a/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMap.m
+++ b/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMap.m
@@ -25,10 +25,10 @@
 #import <objc/runtime.h>
 
 #ifdef HAVE_GOOGLE_MAPS_UTILS
-#import <Google-Maps-iOS-Utils/GMUKMLParser.h>
-#import <Google-Maps-iOS-Utils/GMUPlacemark.h>
-#import <Google-Maps-iOS-Utils/GMUPoint.h>
-#import <Google-Maps-iOS-Utils/GMUGeometryRenderer.h>
+#import "GMUKMLParser.h"
+#import "GMUPlacemark.h"
+#import "GMUPoint.h"
+#import "GMUGeometryRenderer.h"
 #define REQUIRES_GOOGLE_MAPS_UTILS(feature) do {} while (0)
 #else
 #define GMUKMLParser void
diff --git a/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMapHeatmap.h b/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMapHeatmap.h
index 20faf23..eacaefc 100644
--- a/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMapHeatmap.h
+++ b/node_modules/react-native-maps/ios/AirGoogleMaps/AIRGoogleMapHeatmap.h
@@ -4,7 +4,7 @@
 //  Created by David Cako on 29 April 2018.
 //
 
-#import <Google-Maps-iOS-Utils/GMUHeatmapTileLayer.h>
+#import "GMUHeatmapTileLayer.h"
 
 @interface AIRGoogleMapHeatmap : UIView
 

```

**Comment by IroriStudio** [comment]
same isue

**Comment by marquina-abreu** [comment]
same issue

**Comment by Simon-TechForm** [comment]
Hello everyone 👋

I've just released a new beta version that should fix this (see: [v1.5.1-beta.3](https://github.com/react-native-maps/react-native-maps/releases/tag/v1.5.1-beta.3)). Please check it out, and if doesn't solve the issue, feel free to open a new issue!
